memote/suite/tests/test_biomass.py

```diff
@@ -242,21 +242,41 @@ def test_gam_in_biomass(model, reaction_id):
           format_type="raw", data=dict(), message=dict(), metric=dict())
 def test_fast_growth_default(model, reaction_id):
     """
-    Expect the predicted growth rate for each BOF to be below 10.3972.
+    Expect the predicted growth rate for each BOF to be below 2.81.
 
     The growth rate of a metabolic model should not be faster than that of the
-    fastest growing organism. This is based on lowest doubling time reported
-    here:
-    http://www.pnnl.gov/science/highlights/highlight.asp?id=879
+    fastest growing organism. This is based on a doubling time of Vibrio
+    natriegens to be 14.8 minutes as reported by: Henry H. Lee, Nili Ostrov,
+    Brandon G. Wong, Michaela A. Gold, Ahmad S. Khalil, George M. Church in
+    https://www.biorxiv.org/content/biorxiv/early/2016/06/12/058487.full.pdf
+
+    The calculation ln(2)/(14.8/60) ~ 2.81 yields the corresponding growth
+    rate.
+
+    Implementation:
+    Calculate the solution of FBA with the biomass reaction set as objective
+    function and a model's default constraints. Then check if the objective
+    flux is higher than 2.81.
```

Comment:
```suggestion
    value is higher than 2.81.
```
or replace 'objective value' with 'growth rate'.